lib/bolt/inventory/group.rb

```diff
@@ -130,6 +134,7 @@ def data_merge(data1, data2)
           # with the value meant to replace it
           'vars'   => data1['vars'].merge(data2['vars']),
           'facts'  => Bolt::Util.deep_merge(data1['facts'], data2['facts']),
+          'features' => Set.new(data1['features'] + data2['features']).to_a,
```

Comment:
This can just be `data1['features'] | data2['features']`

Reply:
Thanks! Updated.